azkaban-common/src/main/java/azkaban/server/session/SessionCache.java

```diff
@@ -53,12 +57,26 @@
   public SessionCache(final Props props) {
     this.effectiveSessionTimeToLive = props.getLong(ConfigurationKeys.SESSION_TIME_TO_LIVE,
         DEFAULT_SESSION_TIME_TO_LIVE);
+
+    Long maxNumberOfSessions = null;
+    try {
+      maxNumberOfSessions = props.getLong(ConfigurationKeys.MAX_SESSION_NUMBER_PER_IP_PER_USER);
+    } catch (final UndefinedPropertyException exception) {
+      maxNumberOfSessions = null;
```

Comment:
This variable is null already. Should we replace this line with a log message?

Reply:
thanks, removed the initializer.